### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,24 +7,35 @@ php:
...
     - php: 5.3
       env: TRANSPORT=doctrine_dbal COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.3
       env: TRANSPORT=jackrabbit COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.6
-      env: TRANSPORT=doctrine_dbal SYMFONY="3"
+    - php: 7.0
+      env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="2.7"
+    - php: 7.0
+      env: TRANSPORT=jackrabbit SYMFONY_VERSION="2.7"
```
Comment (lsmith77) [inline .travis.yml]: maybe we should remove this?

File: .travis.yml [hunk 2]
```diff
@@ -9,24 +7,35 @@ php:
...
+    - php: 7.0
+      env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="2.8"
+    - php: 7.0
+      env: TRANSPORT=jackrabbit SYMFONY_VERSION="2.8"
```
Comment (lsmith77) [inline .travis.yml]: maybe we should remove this?

**Comment by lsmith77** [comment]
@dbu .. we should merge this before tagging 1.3

File: composer.json [hunk 3]
```diff
@@ -11,32 +11,33 @@
...
-        "jackalope/jackalope-doctrine-dbal": "~1.1",
+        "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "jackalope/jackalope-jackrabbit": "~1.0"
     },
     "conflict": {
         "jackalope/jackalope-doctrine-dbal": "<1.1.5"
     },
     "autoload": {
-        "psr-0": { "Doctrine\\ODM\\PHPCR": "lib/" }
+        "psr-4": { "Doctrine\\ODM\\PHPCR\\": "lib/Doctrine/ODM/PHPCR" }
```
Comment (lsmith77) [inline composer.json]: we need to verify if this is correct

File: .travis.yml (hunk 2 above)
Comment (dbu) [inline .travis.yml]: do you mean to remove it to reduce the number of builds? would it make sense to do symfony 2.7 with dbal and 2.8 with jackrabbit?

Comment (lsmith77) [inline .travis.yml]: yeah .. to reduce the number of builds ..

File: composer.json (hunk 3 above)
Comment (dbu) [inline composer.json]: oh, actually we should probably have moved the code up to lib/ as well and get rid of the extra folders.

**Pull request merged**